TIGLViewer/src/TIGLViewerSettingsDialog.cpp

```diff
@@ -113,13 +137,34 @@ void TIGLViewerSettingsDialog::updateEntries()
     sliderTriangulationAccuracy->setValue(triaVal);
 
     _bgcolor = _settings.BGColor();
+    _shapecolor = _settings.shapeColor();
+    _shapesymmetrycolor = _settings.shapeSymmetryColor();
+    _material = _settings.defaultMaterial();
+
     updateBGColorButton();
+    updateShapeColorButton();
+    updateShapeSymmetryColorButton();
 
     debugBopCB->setChecked(_settings.debugBooleanOperations());
     enumerateFaceCB->setChecked(_settings.enumerateFaces());
     numUIsoLinesSB->setValue(_settings.numFaceUIsosForDisplay());
     numVIsoLinesSB->setValue(_settings.numFaceVIsosForDisplay());
     cbDrawFaceBoundaries->setChecked(_settings.drawFaceBoundaries());
+
+    QMapIterator<QString, Graphic3d_NameOfMaterial> i(tiglMaterials::materialMap);
+    QStringList items;
+    int activeItem = 0;
+    int idx = 0;
+    while (i.hasNext()) {
+        i.next();
+        items << i.key();
+        if (i.value() == _material)
```

Comment:
This loop could be simplified. Better to separate filling the list and finding the index.

The index could be found after populating the list via:
```cpp
int activeItem = items.indexOf(_material);
```

Reply:
Its not possible like this, because _material is the enum value of the Material and not its String name.

Reply:
In this case, let is as it is.